pa_matchbatch.sh

```diff
@@ -72,17 +72,58 @@ getComponentRevMap() {
 	popd
 }
 
-# TODO: more robust version matches?
-# if version file present, use it, otherwise calculate
-# only override versions you care about
-# repo:fix_version, 1 per line
+# Find the fixVersion for a project in this release
 #
+# - If an overrides file was supplied, use the version number from that file
+# - Otherwise look for common version files
+# - Otherwise try to find the most recent version update in the git log
 getFixVerFor() {
-	if [[ -f  ${OVERRIDE_PATH} ]]; then
-		override=$(grep ${1}  ${OVERRIDE_PATH} | cut -d: -f2)
-		[[ -n ${override} ]] && { echo ${override}; return; }
+    # If an overrides file was supplied, look at it for a version number first
```

Comment:
NIT: indenting of the first two possibilities is inconsistent with the other options

Reply:
Ah, this file uses tabs. I'll add a modeline and update